### PR TITLE
Type check with CI

### PR DESCRIPTION
**Pull request by Funth0mas**
Add mypy type check to `tox`, but ignore errorlevel for now.

### CODE REVIEW

**Comment by Funth0mas** [comment]
retry

**Comment by Funth0mas** [comment]
Okay the mypy runs on travis with py 3.6, see [trace from line 500](https://travis-ci.org/ebroecker/canmatrix/jobs/501993036). The build is (for now) passed regardless of issues found by mypy.

File: tox.ini [hunk 1]
```diff
@@ -33,3 +33,11 @@ commands=
 [testenv:old_tests]
 commands=
     ./test.sh
+
+[testenv:mypy]
+description = type check
+basetypthon = python3.6
```
Comment (altendky) [inline tox.ini]:
```suggestion
basepython = python3.6
```

**Comment by Funth0mas** [comment]
The minus sign before `python` in tox.ini means "ignore errorlevel".
I found no `allow_failures` for travis (like used for appveyor). Obviously it would be cleaner solution.

**Comment by Funth0mas** [comment]
Hm seems that I'm blind... I'll try to use `allow_failures`

**Pull request merged**